aspnetcore/includes/razor.md

```diff
@@ -1 +1,5 @@
-This tutorial teaches ASP.NET Core MVC with controllers and views. Razor Pages is a new alternative in ASP.NET Core 2.0, a page-based programming model that makes building web UI easier and more productive. We recommend you try the Razor Pages tutorial before the MVC version. For a Razor Pages version of this tutorial, see [Razor Pages](xref:mvc/razor-pages/index). 
+This tutorial teaches ASP.NET Core MVC with controllers and views using [Visual Studio 2017](https://www.visualstudio.com/). Razor Pages is a new alternative in ASP.NET Core 2.0, a page-based programming model that makes building web UI easier and more productive. We recommend you try the [Razor Pages](xref:mvc/razor-pages/index) tutorial before the MVC version. The Razor Pages tutorial:
```

Comment:
- I recommend linking here instead: https://www.visualstudio.com/vs/
- It feels like everything related to Razor Pages after the 1st sentence should go into a Note widget.